.eslintrc.js

```diff
@@ -15,6 +16,8 @@ module.exports = {
     '@typescript-eslint/no-namespace': 'off',
     '@typescript-eslint/explicit-member-accessibility': 'warn',
     '@typescript-eslint/interface-name-prefix': 'off',
+    //'@typescript-eslint/ban-types': 'off',
```

Comment:
These changes are great! Think it's best to delete the commented out lines rather than comment them out please.